frontend/src/views/SignUp.vue

```diff
@@ -4,7 +4,9 @@
     color="light"
     tile>
     <v-container>
-      <v-form @submit.prevent="signup">
+      <v-form
+        @submit.prevent="signup"
+        v-model="valid">

```

Comment:
Something is wrong with your rules. The `Create an Account` button should only because enabled when the email + password + username are entered. Right now you just need to fill 1 of these 3.